src/main.rs

```diff
@@ -348,6 +330,41 @@ fn find_ram_in_linker_script(linker_script: &str) -> Option<MemoryEntry> {
     None
 }
 
+fn arithmetic_op(line: &str) -> u64 {
+    let segments: Vec<&str> = line.split('+').map(|s| s.trim().trim_end()).collect();
```

Comment:
The name of the function `arithmetic_op` makes me assume that it works for all arithmetic operations. But as far as I see it only works, and it is meant to only work, with `+`. Can you please clarify this in the name of the function, or at least the doc-comment?

Reply:
I will change the name of the function, as it needs to be more comprehensive.